doc/data_workflow.rst

```diff
@@ -0,0 +1,102 @@
+..
+  SPDX-FileCopyrightText: 2021 The PyPSA meets Earth authors
+
+  SPDX-License-Identifier: CC-BY-4.0
+
+.. _data_workflow:
+
+##########################################
+Data used by the model
+##########################################
+
+PyPSA-Earth has the flexible scope when a modeling domain can be selected from the whole Earth to any subregion. The modeling workflow includes data management workflow which is focused on open data collection, modification, prediction and validation. The developed data processing approaches aimed to provide model-ready data with temporal and spatial resolution as high as it's needed by a specific modeling task.
+
+.. _data_management_strategy:
+
+Data management strategy
+===================================
+
+Here we'll look into architecture of the data workflow while practical hand-ons are given in the Tutorial section.
+
+**1. Grid topology data**
+OpenStreetMap `OSM <https://www.openstreetmap.org/>`_ data are used to build power grid topology model. OSM is the biggest crowd-sourced collection of geographic information, which is daily updated and includes geolocation references. The OSM data are being loaded by `download_osm_data` and cleaned by `clean_osm_data` rules, respectively.
+
+**2. Climate data**
+The climate data processing is provided by `atlite <https://atlite.readthedocs.io/en/latest/>`_ package. It extracts all the required whether and climate data to generate the time series of renewable potential by g`enerate_renewable_profiles` rule.
+
+The main data source on climate variables is `ERA5 reanalysis <https://rmets.onlinelibrary.wiley.com/doi/10.1002/qj.3803>`_.
+
+**3. General data**
+There are a number datasets applied in PyPSA-Earth to build a realistic model. Original datasets are stored in the `pypsa-earth/data/` folder.
+
+Currently we are using the following resources.
+
+- *environmental*
```

Comment:
The *environmental* may be formatted using titles in rst format. See examples here: https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#sections

Reply:
Done!